frontend/ui/data/keyboardlayouts/ar_AA_keyboard.lua

```diff
@@ -0,0 +1,165 @@
+            { _D_,        _d_,    "%",    "5",    "ִ ",    yaa,    "-",    "5",    "Î",    "î",    "Ğ",    "ğ", },
+            { _F_,        _f_,    "^",    "6",    "ֹ",    baa,      _eq,    "6",    "Ï",    "ï",    "Ć",    "ć", },
+            { _G_,        _g_,    ":",    ";",    "ְ ",    lam,      "Ү",    "ү",    "Ô",    "ô",    "Č",    "č", },
+            { _H_,        _h_,    '"',    "'",    "ֵ ",    alef,        "Ұ",    "ұ",    "Œ",    "œ",    "Đ",    "đ", },
+            { _J_,        _j_,    "{",    "[",    "ֶ ",    taa,       "Қ",    "қ",    "Ù",    "ù",    "Š",    "š", },
+            { _K_,        _k_,    "}",    "]",    "ַ ",   nun,      "Ж",    "ж",    "Û",    "û",    "Ž",    "ž", },
+            { _L_,        _l_,    "_",    "-",    "ָ ",    meem,       "Э",    "э",    "Ÿ",    "ÿ",    "Ő",    "ő", },
+            { ";",        _l_,    "_",    "-",    "ָ ",    kaf,       "Э",    "э",    "Ÿ",    "ÿ",    "Ő",    "ő", },
+            { "'",        _l_,    "_",    "-",    "ָ ",    tah,       "Э",    "э",    "Ÿ",    "ÿ",    "Ő",    "ő", },
+        },
+        -- third row
+        {  --  1           2       3       4       5       6          7       8       9       10      11      12
+            { label = "بَدِّل",
+              icon = "resources/icons/appbar.arrow.shift.png",
+              width = 1.5
+            },
+            { _Z_,        _z_,    "&",    "7",    "׃",    thaal,     ":",    "7",    "Á",    "á",    "Ű",    "ű", },
+            { _X_,        _x_,    "*",    "8",    "׀",    hamza,    ";",    "8",    "Ø",    "ø",    "Ã",    "ã", },
+            { _C_,        _c_,    "£",    "9",    "ׄ ",    wawhamza,      "'",    "9",    "Í",    "í",    "Þ",    "þ", },
+            { _V_,        _v_,    "<",    com,    "ׅ ",    raa,       "Ө",    "ө",    "Ñ",    "ñ",    "Ý",    "ý", },
+            { _B_,        _b_,    ">",    prd,    "־",    laa,       "Һ",    "һ",    "Ó",    "ó",    "†",    "‡", },
+            { _N_,        _n_,    "‘",    "↑",    "ֿ ",    alefmaqsoura,      "Б",    "б",    "Ú",    "ú",    "–",    "—", },
+            { _M_,        _m_,    "’",    "↓",    "ֽ ",    taamarbouta,   "Ю",    "ю",    "Ç",    "ç",    "…",    "¨", },
+            { ",",        ",",    "’",    "↓",    "ֽ ",    waw,   "Ю",    "ю",    "Ç",    "ç",    "…",    "¨", },
+            { ".",        ".",    "’",    "↓",    "ֽ ",    zay,   "Ю",    "ю",    "Ç",    "ç",    "…",    "¨", },
+            { "/",        "/",    "’",    "↓",    "ֽ ",    thaa,   "Ю",    "ю",    "Ç",    "ç",    "…",    "¨", },
+            { label = "مَسح",
+              icon = "resources/icons/appbar.clear.reflect.horizontal.png",
+              width = 1.5
+            },
+        },
+        -- fourth row
+        {
+            { "Sym",     "Sym",  "ABC",  "ABC",  "Sym",  "رمز",    "رمز",  "رمز",  "Sym",  "Sym",  "ABC",  "ABC",
+              width = 1.5},
```

Comment:
You could try reducing this width to = 1 to not have these keys overflow the keyboard :)